### DISCUSSION TITLE
Watching from websites other than YouTube

### DISCUSSION THREAD

**Discussion by tahadnan** [mov-cli/mov-cli#324] [Q&A] [answered]
Hello dear people, I was wondering if I could watch media from websites other than YouTube , thanks in advance.

**Comment by THEGOLDENPRO** [comment, marked as answer]
Yeah that is possible if there's a plugin that can stream from that site in question. You can see the list of third-party plugins [here](https://github.com/topics/mov-cli-plugin).
